### ISSUE TITLE
NPCs being knocked out by the piano use "fire" voice lines instead of "crushed"

### ISSUE THREAD

**Issue by Ocean-Minnow** [glacier-modding/H3-Community-Patches#132]
The fix for this is already made, I just need to put it in here.

**Comment by AnthonyFuller** [comment]
Could use other lines of them screaming for them.

**Comment by Ocean-Minnow** [comment]
Yeah, it's not so much a matter of finding lines, I just need to put them into the SDEF and it hasn't been a huge priority, since the "fire" lines work well enough if it isn't a S2+ voice (and I don't think pianos appear in S2?)
